test/org/sagebionetworks/bridge/services/ScheduledActivityServiceDuplicateTest.java

```diff
@@ -270,24 +269,22 @@ public void testExistingDuplicationScenario() throws Exception {
         verify(schedulePlanService).getSchedulePlans(any(), any());
         
         allWithinQueryWindow(activities, context);
-        //assertEquals(16, activities.size());
-        assertEquals(1, filterByGuid(activities, "bea8fd5d-7622-451f-a727-f9e37f00e1be:2016-06-30T19:43:07.951").size());
-        assertEquals(1, filterByGuid(activities, "6966c3d7-0949-43a8-804e-efc25d0f83e2:2016-06-30T19:43:07.951").size());
-        assertEquals(1, filterByGuid(activities, "79cf1788-a087-4fa3-92e4-92e43d9699a7:2016-06-30T19:43:07.951").size());
-        // NOTE: this task has reset because the timestamp is different
-        // assertNotNull(filterByLabel(activities, "Training Session 1").get(0).getStartedOn());
+        assertEquals(0, filterByGuid(activities, "bea8fd5d-7622-451f-a727-f9e37f00e1be:2016-06-30T12:43:07.951").size());
```

Comment:
The persisted tasks include a task with this GUID that has been finished, so it should be removed from the results. Another test executes without any of the persisted tasks, to verify that it is returned in that case. There's also a duplication of this task so when it is returned, there should only be 1.